RDKShell/RDKShell.cpp

```diff
@@ -6578,49 +6578,103 @@ namespace WPEFramework {
                         returnResponse(status);
                     }
                 }
+		// Container mode hibernate functionality handling
+		auto ociContainerPlugin = getOCIContainerPlugin();
+                if (!ociContainerPlugin)
+                {
+                    response["message"] = "OCIContainer plugin initialisation failed";
+                    returnResponse(false);
```

Comment:
We don't use/enable OCIContainer plugin on some builds, this will break the hibernate feature.